src/components/dashboard/shared/VerticalTimeline.js

```diff
@@ -51,14 +69,18 @@ export default class VerticalTimeline extends Component {
   }
 
   render() {
-    const { items } = this.props;
+    const { items, viewCount } = this.props;
     const { displayCount } = this.state;
     const sortedItems = this.sortItems(items);
 
     return (
       <div className="vertical-timeline">
         {sortedItems.slice(0, displayCount).map((item, index) => this.renderItem(item, index))}
-        {(sortedItems.length > displayCount) && this.renderViewMore()}
+
+        <p className="vertical-timeline__view-more-less">
+          {(sortedItems.length > displayCount) && this.renderViewMore()}
+          {(displayCount > viewCount) && this.renderViewLess(sortedItems.length > displayCount)}
```

Comment:
For the same reason, I'd change this line to `{(displayCount > initialDisplayCount) && this.renderViewLess(sortedItems.length > displayCount)}` and add the prop name to line 72.

You will need to fix your tests after making these changes.